openpype/hosts/aftereffects/api/extension/jsx/hostscript.jsx

```diff
@@ -322,7 +325,7 @@ function importFile(path, item_name, import_options){
 }
 
 
-function importFileWithDialog(path, item_name){
+function importFileWithDialog(path, item_name, fps){
```

Comment:
fps should be added in an `import_options` object argument, not as standalone argument